MAUI/Calendar/date-restrictions.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Date Restrictions in MAUI Calendar widget | Syncfusion
+description: Learn here all about Date restrictions feature of Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Date Restrictions in MAUI Calendar (SfCalendar)
+
+## Minimum DateTime
+The [MinimumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_MinimumDate) property will restrict the backward navigation of date selection and also you cannot able to swipe beyound the minimum date range. The dates before the minimum date time is considered as disabled date.
```

Comment:
The [MinimumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_MinimumDate) property will restrict the backward navigation of date selection and also you cannot swipe beyond the minimum date range.

Reply:
Comments modified